src/seeders/20190826080729-create-role-permissions-seeders.js

```diff
@@ -0,0 +1,391 @@
+/* eslint-disable no-unused-vars */
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('RolePermissions', [
```

Comment:
I don't think this style of seeding the `RolePermission` table is descriptive enough. It's a bit error prone. How do we know what permissions are assigned to what roles by looking at the code. I think this could be more declarative.